randovania/data/json_data/prime1.txt

```diff
@@ -2402,8 +2477,12 @@ Asset id: 1681720207
       After Main Quarry Barrier Deactivated
   > Pickup (Missile)
       All of the following:
-          Wave Beam and Scan Visor and Space Jump Boots and Morph Ball and Spider Ball
+          Wave Beam and Scan Visor
           Thermal Visor or Invisible Objects (Beginner)
+          Any of the following:
+              Space Jump Boots and L-Jump (Beginner) and Slope Jump (Intermediate) and Standable Terrain (Beginner)
+              Morph Ball and Spider Ball
+          Space Jump Boots or Slope Jump (Intermediate)
```

Comment:
This line seems wrong. It's part of the all of the following grouping, so it invalidates the Spider path.

Reply:
I went and double checked, and I'm fairly sure I got this right, but here's my thinking so you can hopefully tell me where I went wrong.

Scan Visor - Required to scan terminal
Wave Beam - to activate conduit
Thermal Visor || Invisible Objects - to know where conduit is
Space Jump || Slope Jump(Intermediate) - to get to the part with the terminal/crane. There are some rocky jumps that require slope jumps just to get up to the section with the crane if you're NSJ.

The OR block, which can select either spider and morph or a bunch of tricks, is just for getting from the base of the crane to the pickup.

From the base of the crane, you can slope double jump off the slope of the terminal housing to stand on a lip at the top of the crane. From there you can double L-Jump to the crane's load, thus skipping the requirement for morph ball or spider ball.

Reply:
I just double checked and the Space Jump or Slope Jump path is part of the "All of the Following" so it does invalidate the morph/spider path
![image](https://user-images.githubusercontent.com/38679103/99934059-e0e65580-2d2a-11eb-94d8-d27434fd535d.png)

Reply:
As it should be - there is no way to traverse the path without one of the two (obviously barring any tricks I haven't tried). 

This pair of jumps is the reason for that requirement
![image](https://user-images.githubusercontent.com/13577614/99935353-c063ce80-2d1d-11eb-9cb2-fcf432683fad.png)


It is arranged in a way that is graphically quite odd, I'll try and organize it in a way that looks a little better.

Reply:
ah i see it then. remove the space jump and slope jump requirements from the "Any of the following part." that isnt needed to be stated since its already covered with the "OR"

Reply:
They should already be gone since it'll now be quarry access -> top platform -> missiles instead of just quarry access -> missiles.